packages/bpk-component-breakpoint/src/useMediaQuery.ts

```diff
@@ -16,25 +16,24 @@
  * limitations under the License.
  */
 
-import { useState, useEffect } from 'react';
+import { useEffect, useState } from 'react';
+
+const useMediaQuery = (query: string, matchSSR = false): boolean => {
+  const isClient = typeof window !== 'undefined' && !!window.matchMedia;
 
-const useMediaQuery = (query: string): boolean => {
   const [matches, setMatches] = useState(
-    window.matchMedia ? window.matchMedia(query).matches : false,
+    isClient ? window.matchMedia(query).matches : matchSSR,
   );
 
   useEffect(() => {
-    if (window.matchMedia) {
-      const media = window.matchMedia(query);
+    const media = window.matchMedia(query);
+    setMatches(media.matches);
+    const listener = () => {
       setMatches(media.matches);
-      const listener = () => {
-        setMatches(media.matches);
-      };
-      media.addEventListener('change', listener);
-      return () => media.removeEventListener('change', listener);
-    }
-    return undefined;
-  }, [query]);
+    };
+    media.addEventListener('change', listener);
+    return () => media.removeEventListener('change', listener);
+  }, [query, isClient]);
```

Comment:
we don't have `isClient` as a dependency anymore so we can remove this?

Reply:
Removed